rogue-thi-app/pages/api/reimanns.js

```diff
@@ -94,7 +103,14 @@ export default async function handler (req, res) {
         }))
       }))
 
-      return translateMeals(mealPlan)
+      const scrapedMeals = await translateMeals(mealPlan)
+
+      // add static meals (no need to translate)
+      scrapedMeals.forEach(day => {
+        day.meals.push(...staticMeals)
```

Comment:
In my opinion we should allow the users to select wether they want to see static reimanns meals, since most people know about their existence and check the app solely for daily meal information

Reply:
That's a good point, but I've heard more and more from students that they forget these dishes and don't have them on their radar because they're not listed. Nevertheless, I will add the filter but would show the dishes by default.